tests/test_logs.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pytest
+from flask import session
+
+from management_app import create_app
+
+@pytest.fixture
+def app():
+  app = create_app({
+      'TESTING': True,
+  })
+  yield app
+
+@pytest.fixture
+def client(app):
+  return app.test_client()
+
+@pytest.fixture
+def mock_login(client):
+  # Enable login auth
+  with client.session_transaction() as session:
+      session['google_id'] = 'TEST_ID_NUM'
```

Comment:
The three test cases look failed. Actually, you don't need `@pytest.fixture` in the `mock_login` because it is a re-usable python function so it is not meant to be called directly.